doc/ConsoleHostSettings.md

```diff
@@ -38,6 +38,9 @@ Settings in the Windows Console Host can be a bit tricky to understand. This is
 |`VirtualTerminalLevel`*    |REG_DWORD              |The level of VT support provided by the Windows Console Host |
 
 *: Only applies to the improved version of the Windows Console Host
+**: WxH stands for Width by Height, it's the fact that things like a Window size

```

Comment:
This is appearing as merged into the previous asterisk footnote. I think you need another newline to make GitHub markdown separate it for real.